app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -240,4 +245,13 @@ open class UserViewModel(
           }
         }
   }
+
+  fun getUsersByIds(userIds: List<String>, onSuccess: (List<User>) -> Unit) {
+    userRepository.fetchUsersByIds(userIds, onSuccess, {})
+  }
+
+  fun getMyContacts(onSuccess: (List<User>) -> Unit) {
+    if (Firebase.auth.uid == null) return
+    userRepository.getContacts(Firebase.auth.uid ?: "", onSuccess, {})
```

Comment:
The OnFailure method is empty for both methods, can we add error logs for debugging purposes?